.rubocop_todo.yml

```diff
@@ -57,6 +63,38 @@ Minitest/EmptyLineBeforeAssertionMethods:
     - 'test/new_relic/agent_test.rb'
     - 'test/new_relic/cli/commands/deployments_test.rb'
 
+# Offense count: 23
+# This cop supports safe autocorrection (--autocorrect).
+Performance/RegexpMatch:
+  Enabled: false
+
+# Offense count: 1
+# This cop supports unsafe autocorrection (--autocorrect-all).
+# Configuration parameters: OnlySumOrWithInitialValue.
+Performance/Sum:
+  Exclude:
+    - 'lib/new_relic/agent/system_info.rb'
+
+# Offense count: 72
+# This cop supports unsafe autocorrection (--autocorrect-all).
+Performance/UnfreezeString:
+  Enabled: false
+
+# Offense count: 21
+# This cop supports safe autocorrection (--autocorrect).
+# Configuration parameters: AllowedVars.
+Style/FetchEnvVar:
+  Enabled: false
+
+# Offense count: 1
+# This cop supports safe autocorrection (--autocorrect).
 Style/RedundantStringEscape:
```

Comment:
To continue Tanna's theme above, let's try to address this one-off too

Reply:
oh huh, looking at that file, i'm pretty sure this is something we can't fix. We're specifically testing the behavior for when there IS an extra \ the end of the string. We should probably just remove this from here and add like a disable cop thing to that line in the test file instead.

Reply:
I'll preserve this info in a rubocop ticket^